<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ITT-MASINENI" failures="1" tests="3" name="Test" time="54.191" errors="0" timestamp="2025-02-04T20:45:26 IST">
  <testcase classname="AutomationExercise.Tests.PositiveTests" name="FailTest" time="0.011">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at AutomationExercise.Tests.PositiveTests.FailTest(PositiveTests.java:143)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- FailTest -->
  <testcase classname="AutomationExercise.Tests.PositiveTests" name="loginWithCorrectCredentials" time="4.457"/>
  <testcase classname="AutomationExercise.Tests.PositiveTests" name="verifyRegisterUser" time="7.923"/>
</testsuite> <!-- Test -->
